pycqed/analysis_v3/tomography_analysis.py

```diff
@@ -929,18 +927,27 @@ def process_tomography_analysis(data_dict, gate_name='CZ', Uideal=None,
         - chi_{gate_name}.{estimation_type} and
             measured_error_{gate_name}.{estimation_type} for estimation_type
             in estimation_types.
+        -  process_name: name of the gate for which the error is estimated.
+         MUST CORRESPOND TO Uideal IF THE LATTER IS PROVIDED, since gate_name
```

Comment:
If this is a requirement, it might make sense to
- add a check whether the requirement is fulfilled
- better explain in what way the correspondence is meant, e.g., where can I look up the names that are expected for certain unitaries. Or do you mean the following? "If both Uideal and process_name are provided, process_name will only be used in the key name for storing the results, and it is up to the user to ensure that process_name is a meaningful description of the unitary Uideal."  --- but this is not what the code currently implements.

Moreover, the reference to the parameter "gate_name" seems to be outdated.

Reply:
I updated the docstring now and also fixed a bug that I realized after reading your comment. Thanks